test/javascript/util/keyboard.js

```diff
@@ -20,7 +20,7 @@ describe('Keyboard util', function() {
   };
 
   afterEach(function() {
-    document.activeElement.blur();
+    if(document.activeElement)document.activeElement.blur();
```

Comment:
```js
if(document.activeElement)document.activeElement.blur();
//                        ^
//                     no space ?
```

You can rebase this change with the previous commit I guess. The PR is clear enough. And use `style: ...`